PizzaAPI/PizzaAPI/Repositories/PizzaRepositorie.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.EntityFrameworkCore;
+using PizzaAPI.DTOS;
+using PizzaAPI.Models;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace PizzaAPI.Repositories
+{
+    public class PizzaRepositorie : IPizzaRepositorie
+    {
+        private readonly DataContext _context;
+
+        public PizzaRepositorie(DataContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<PizzaModel> AddLikesOrDislikes(PizzaDescriptionDto pizza)
+        {
+            var pizzaToUpdate = await _context.Pizzas.FirstOrDefaultAsync(e => e.Id == pizza.Id);
+            if (pizzaToUpdate != null)
```

Comment:
W nowym C# lepiej pisać, żeby uniknąć problemów na nadpisanym operatorem porównania
```suggestion
            if (pizzaToUpdate is not null)
```